settings.gradle.kts

```diff
@@ -69,6 +69,11 @@ include(":utils:core")
 include(":utils:spdx")
 include(":utils:test")
 
+project(":utils:common").name = "common-utils"
+project(":utils:core").name = "core-utils"
+project(":utils:spdx").name = "spdx-utils"
+project(":utils:test").name = "test-utils"
+
```

Comment:
Not sure if it's worth it, but to group inclusion with setting the project name we could do something like

```
listOf("common", "core", "spdx", "test").forEach {
    val path = ":utils:$it"
    include(path)
    project(path).name = "$it-utils"
}
```

(And then maybe use similar code for the clients.)

Reply:
I thought about it as well, but decided not to do something like this because the current way is simple and expressive and not much to write anyway.